doc/source/examples_source/00-basic/03-ptrace.py

```diff
@@ -0,0 +1,145 @@
+session.ensight.case.select("Case 1")
+session.ensight.data_partbuild.end()
+
+
+###############################################################################
+# Setup the View
+# ----------------------------------
+# Set View, Turn on Symmetry, turn off visibility of flow domain.
+
+session.ensight.view_transf.rotate(-109.084335, -1.64276719, 0)
+session.ensight.view_transf.rotate(-2.38553524, 115.462845, 0)
+session.ensight.view_transf.zoom(0.489253074)
+session.ensight.view_transf.look_at(2.4110589, 0.567389309, 0.241451085)
+session.ensight.view_transf.look_from(2.4110589, 0.567389309, 5.69335651)
+
+session.ensight.part.select_all()
+session.ensight.part.modify_begin()
+session.ensight.part.symmetry_type("mirror")
+session.ensight.part.symmetry_mirror_y("ON")
+session.ensight.part.modify_end()
+
+session.ensight.part.select_begin(1)
+session.ensight.part.modify_begin()
+session.ensight.part.visible("OFF")
+session.ensight.part.modify_end()
+
+###############################################################################
+# Create a Particle Trace using the Line Tool Specification
+# ---------------------------------------------------------
+# Using the 3D parts as the parent, with the line tool as the emission type
+# "Momentum" as the vector, and 50 points alone the line as emitter locations.
+
+# .. image:: /_static/03_ptrace_1.png
+
+pt1 = [0.1245, 0.064366, -0.03438]
+pt2 = [1.0018, 0.399756, -0.03258]
+parent_parts = eoutil.parts.select_parts_by_dimension(3)
+npts = 50  # number of emitters
+vector_var = eocore.VARIABLES["Momentum"][0]  # Vector variable to use
+strpart = eoutil.parts.create_particle_trace_from_line(
+    "Streamline",
+    vector_var,
+    point1=pt1,
+    point2=pt2,
+    num_points=npts,
+    source_parts=parent_parts,
+    direction="+/-",
+)
+session.show("image", width=800, height=600)
+
+###############################################################################
+# Change Visual Attributes
+# ----------------------------------------------------------
+#  Modify the attributes of the Streamline for visual clarity
+
+# .. image:: /_static/03_ptrace_2.png
+
+strpart.REPRESENTATION = eonums.TRACE_TUBE
+strpart.WIDTHSCALEFACTOR = 0.012
+strpart.COLORBYPALETTE = "Momentum"
+session.show("image", width=800, height=600)
+
+###############################################################################
+# Animate the Streamlines
+# ----------------------------------------------------------
+#  Turn OFF the streamlines (to see the animate under)
+#  Turn ON the animate streamlines.
+#  Change to Sphere representation, size, and adjust speed and length.
+
+# .. image:: /_static/03_ptrace_3.png
+
+strpart.VISIBLE = False
+strpart.ANIMATE = True
+eocore.HEADTYPE = eonums.ATRACE_HEAD_SPHERE
+eocore.HEADSCALE = 0.03
+eocore.PARTICLETIME = 2.0
+eocore.DELTATIME = 0.065
+eocore.MULTIPLEPULSES = True
+session.show("image", width=800, height=600)
+
+session.close()
```

Comment:
@kecolburn sorry, last comment. I checked the rendered page and I can see the example hasn't got a thumbnail.

You can see at the bottom of any example there's a commented line like this:

# sphinx_gallery_thumbnail_path = '_static/02_calc_4.png'

Could you please add an equivalent one for this example? Personally I'd choose 03_ptrace_2.png as thumbnail, but I'll leave it to you